src/types.tsx

```diff
@@ -397,6 +397,8 @@ export interface ScreenStackProps extends ViewProps {
    * A callback that gets called when the current screen finishes its transition.
    */
   onFinishTransitioning?: (e: NativeSyntheticEvent<TargetedEvent>) => void;
+  stackRef?: React.MutableRefObject<React.Ref<View>>;
+  ref?: React.MutableRefObject<React.Ref<View>>;
```

Comment:
Let's maybe put a comment above `stackRef` and `ref` what is the difference between them